public/app/plugins/datasource/cloud-monitoring/components/Metrics.tsx

```diff
@@ -39,6 +40,21 @@ export function Metrics(props: Props) {
     projectName: null,
   });
 
+  const theme = useTheme2();
+  const styles = getSelectStyles(theme);
+  const getStyle = stylesFactory((theme: GrafanaTheme2) => {
```

Comment:
stylesFactory is deprecated. Can you use the `useStyles` hook instead? Also, please break this out into a separate `getStyles` function so that we follow the same pattern as in other components.